package.json

```diff
@@ -36,6 +37,7 @@
         "bootstrap": "4.3.1",
         "chart.js": "2.8.0",
         "core-js": "3.1.3",
+        "franc": "^4.1.0",
```

Comment:
Would [`franc-min`](https://github.com/wooorm/franc/tree/master/packages/franc-min) be sufficient?

Reply:
unfortunately, there is no TypeScript definition package for franc-min, but there is one for franc

Reply:
How large ist the franc package? Our client app is already quite large with about 2 MB, so we don’t want to add too large packages to the app

Reply:
`franc`: https://bundlephobia.com/result?p=franc@4.1.0
`franc-min`: https://bundlephobia.com/result?p=franc-min@4.1.0

Reply:
Then i definitely prefer franc-min. Should the type definitions not work for both libs? I guess API wise they are the same... we could even check if we can get a version with only English and German running. This might be even smaller...

Reply:
I swapped franc with franc-min